src/styles/components/user.scss

```diff
@@ -0,0 +1,10 @@
+.navbar-expand-lg{
+  padding:10px 25px;
+  justify-content: space-between;
```

Comment:
Do it have postCSS? (with autoprefixer for IE)

Reply:
No. In case of necessary I can add full browser support. As for styles in task was written that styles are on my taste, and according statistics the part of IE is lower than 5% so I didnt use autoprefixer.